lib/apiservers/engine/backends/portlayer/container_proxy.go

```diff
@@ -259,8 +269,29 @@ func (c *ContainerProxy) AddVolumesToContainer(handle string, config types.Conta
 	return handle, nil
 }
 
-// CommitContainerHandle() commits any changes to container handle.
+// AddPortsToContainer adds ports to a container, referenced by handle.
+// If an error is return, the returned handle should not be used.
 //
+// returns:
+//	modified handle
+func (c *ContainerProxy) AddPortsToContainer(handle string, config types.ContainerCreateConfig) (string, error) {
```

Comment:
I just followed the other naming convention in that file. I could change it, what should we use?